.github/workflows/ci.yml

```diff
@@ -25,14 +29,6 @@ jobs:
       run: node Makefile checkRuleFiles
```

Comment:
`node Makefile lint` and `node Makefile checkRuleFiles` include checking files inside `docs`. If the intent is to run these jobs only when there are changes outside `docs`, then these checks will be missing when all the changes are in `docs`.

Reply:
`node Makefile lint` doesn't seem to check the `docs/**` directory
https://github.com/eslint/eslint/blob/337f7ed96131d873be7ae6b010739476d0ad15e9/Makefile.js#LL537

Reply:
It includes linting markdown files inside `docs`:

https://github.com/eslint/eslint/blob/a061527a0332f0edf559acfc2902a327cae098d9/Makefile.js#L545-L549